src/main/java/io/kestra/plugin/gitlab/issues/CreateIssue.java

```diff
@@ -0,0 +1,117 @@
+package io.kestra.plugin.gitlab.issues;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.kestra.core.http.HttpRequest;
+import io.kestra.core.http.HttpResponse;
+import io.kestra.core.http.client.HttpClient;
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.gitlab.AbstractGitLabTask;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Create a GitLab issue",
+    description = "Create a new issue in a GitLab project"
+)
+@Plugin(examples = {
+    @Example(
+        title = "Create an issue in a GitLab project using a [project access token](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html).",
+        full = true,
+        code = """
+            id: gitlab_create_issue
+            namespace: company.team
+
+            tasks:
+              - id: create_issue
+                type: io.kestra.plugin.gitlab.issues.CreateIssue
+                url: https://gitlab.com
```

Comment:
Add the URI in it ("/api/...") and let the user choose but let the field not mandatory